applications/dashboard/modules/class.headmodule.php

```diff
@@ -137,9 +137,10 @@ public function addTag($tag, $properties, $content = null, $index = null) {
          *  - numeric: This will be the script's sort.
          *  - string: This will hint the script (inline will inline the file in the page.
          *  - array: An array of options (ex. sort, hint, version).
+         * @param string $content The content of the script tag.
          *
          */
-        public function addScript($src, $type = 'text/javascript', $addVersion = true, $options = []) {
+        public function addScript($src, $type = 'text/javascript', $addVersion = true, $options = [], $content = null) {
```

Comment:
I think you can just call addScript with `['content' => '...']` in the $options parameter.